### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by dn1470**
Typescript skeleton pages

navigation.tst added (React Routes file)

### CODE REVIEW

**Comment by dn1470** [comment]
I am not sure how to pass the first Frontend CI checks. The only difference is I installed "react-router-dom" by using "npm install react-router-don".

**Comment by thefishua** [comment]
> I am not sure how to pass the first Frontend CI checks. The only difference is I installed "react-router-dom" by using "npm install react-router-don".

Its prettier basically the code style is bad

File: terraform/aws_only/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+resource "aws_kms_key" "ecr_kms" {
```
Comment (The-TT-Hacker) [inline terraform/aws_only/ecr.tf:2]: why is this file here?

File: frontend/src/navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { useEffect, useState } from 'react';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import Home from './pages/home';
+import GitHublogin from './pages/githublogin';
+
+export function Navigation() {
+  return (
+    <Routes>
+      <Route path="/home" element={<Home />} />
+      <Route path="/login" element={<GitHublogin />} />
```
Comment (The-TT-Hacker) [inline frontend/src/navigation.tsx]: GitHublogin should be GitHubLogin

File: .vscode/targets.log [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+# automatic
+<D = $(patsubst %/,%,$(dir $<))
+# automatic
+?F = $(notdir $?)
+# environment
+WSLENV = VSCODE_WSL_EXT_LOCATION/up
+# default
+.SHELLFLAGS := -c
+# environment
+NVM_INC = /home/youngju/.nvm/versions/node/v16.15.1/include/node
```
Comment (The-TT-Hacker) [inline .vscode/targets.log]: lets not add these files, they have absolute paths from your system

**Comment by The-TT-Hacker** [comment]
> I am not sure how to pass the first Frontend CI checks. The only difference is I installed "react-router-dom" by using "npm install react-router-don".

run `npm run fix`
